pom.xml

```diff
@@ -9,68 +9,116 @@
 	License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS 
 	OF ANY KIND, either express or implied. See the License for the specific 
 	language governing permissions and limitations under the License. -->
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
 
 	<modelVersion>4.0.0</modelVersion>
 	<parent>
 		<groupId>org.apache.sling</groupId>
-		<artifactId>sling</artifactId>
-		<version>30</version>
+		<artifactId>sling-bundle-parent</artifactId>
+		<version>52</version>
 		<relativePath />
 	</parent>
 
 	<artifactId>org.apache.sling.commons.json</artifactId>
-	<packaging>bundle</packaging>
 	<version>2.0.21-SNAPSHOT</version>
 
 	<name>Apache Sling JSON Library</name>
 	<description>Apache Sling JSON Library</description>
 
 	<scm>
-		<connection>scm:svn:http://svn.apache.org/repos/asf/sling/trunk/bundles/commons/json</connection>
-		<developerConnection>scm:svn:https://svn.apache.org/repos/asf/sling/trunk/bundles/commons/json</developerConnection>
-		<url>http://svn.apache.org/viewvc/sling/trunk/bundles/commons/json</url>
+		<connection>scm:git:https://github.com/apache/sling-org-apache-sling-commons-json</connection>
+		<developerConnection>scm:git:https://github.com/apache/sling-org-apache-sling-commons-json</developerConnection>
+		<url>https://github.com/apache/sling-org-apache-sling-commons-json</url>
 	</scm>
 
+	<properties>
```

Comment:
These are not needed. If you need to override the default Java version, set the `sling.java.version` property to `8`.